lib/application/widgets/theme/negative_button.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:memo/application/theme/theme_controller.dart';
+
+/// Negative alternative to [ElevatedButton].
+///
+/// Used in scenarios where the user action may be destructive
+class NegativeButton extends HookWidget {
+  const NegativeButton({required this.onPressed, required this.child, Key? key}) : super(key: key);
```

Comment:
Just like I suggested before, I think destructive would be better - but really personal, I'd say negative is good as well.

My only rationale with preferring the former is that destructive is not necessarily a negative behavior - meaning that destroying something is completely acceptable, although requires enough emphasis.

Reply:
Updated do destructive